src/core/UserIconGroup/UserIconGroup.tsx

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import cx from 'classnames';
+import { CommonProps } from '../utils/props';
+
+export type UserIconGroupProps = {
+  /**
+   * UserIconGroup max number of icons unstacked
+   * @default 5
+   */
+  maxIcons?: number;
+  /**
+   * UserIconGroup is group needs to be stacked
+   * @default 'true'
+   */
+  stacked?: boolean;
+  /**
+   * UserIconGroup type
+   * @default 'false'
+   */
+  animated?: boolean;
+  /**
+   * UserIconGroup type
+   */
+  iconSize: 'small' | 'medium' | 'large' | 'x-large';
+  /**
+   * User Icons in the UserIconGroup.
+   */
+  children: React.ReactNode;
+
+  /**
+   * User count icon props, eg. onClick
```

Comment:
It is `e.g.`. 
No need empty line above too.

Reply:
removed empty line